test/Typesense/SearchClient.spec.js

```diff
@@ -1,6 +1,6 @@
 import chai from 'chai'
 import chaiAsPromised from 'chai-as-promised'
-import {SearchClient as TypesenseSearchClient} from '../../src/Typesense'
+import { SearchClient as TypesenseSearchClient } from '../../src/Typesense'
```

Comment:
I do find these a little confusing since there aren't any conflicts and we appear to be used as an alias. It might be clearer in the future to simply use the actual class name without renaming it so its easier to follow what's being used and tested. 

I've avoided changing any tests as much as possible since that is the only way to determine if the library still functions as intended.